conversation_reconstruction_dataflow/conversation_reconstruction/construct_utils/run_constructor.py

```diff
@@ -0,0 +1,81 @@
+"""
+Copyright 2017 Google Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-------------------------------------------------------------------------------
+
+Run Constructor
+
+Runs wikipedia_revisions_constructor.py with command line arguments for input.
```

Comment:
Good to give a high level idea of what that does here too. Else every reader has to go and read another file.

Reply:
Will update the functionality for each file in README later.